build.gradle

```
@@ -305,6 +330,7 @@ versioner {
 	}
 }
 
+build.dependsOn(asciidoctor)
```

```suggestion
//build.dependsOn(asciidoctor)
```

its quite slow and still not fully releasable thus I would keep `asciidoctor` as an independent task.

---

ok